rpp-charter.md

```diff
@@ -77,13 +83,11 @@ of the chairs with the responsible Area Director:
 * core architecture
 * specifications necessary to implement the core architecture
 * specifications regarding the provisioning of domain names using RPP
-* mappings between RPP and EPP.
+* mappings between RPP and EPP
 
-The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses)
-with consultation from the responsible Area Director.
+The working group may consider work for the provisioning of other types of Internet registries (e.g. IP addresses
```

Comment:
again, same comment, is RPP limited to Internet resources or can it be used for anything requiring a central registry function?